### PR TITLE
Results of lesson 10.

### PR DESCRIPTION
**Pull request by ssaleta**

### CODE REVIEW

**Comment by ssaleta** [comment]
@gradopaw  4 lessons left 😫

File: students/saleta_sebastian/lesson_10_organizing_files/filling_in_the_gaps.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+FILE_NAME = 'spam'
+TXT_EXTENSION = 'txt'
+
+
+def create_test_data():
+    if not os.path.exists(TEST_FILES_DIRECTORY):
+        os.makedirs(TEST_FILES_DIRECTORY)
+
+    # Create files with gaps
+    for i in range(2, 9):
```
Comment (gradopaw) [inline students/saleta_sebastian/lesson_10_organizing_files/filling_in_the_gaps.py]: I have a bit modified your test data:
```for i in random.sample(range(2, 20), 5):``` and as I can see results are wrong 😄

File: students/saleta_sebastian/lesson_10_organizing_files/insert_the_gaps.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        open(os.path.join(TEST_FILES_DIRECTORY,
+                          "{}{:03d}.{}".format(FILE_NAME, i, TXT_EXTENSION)),
+             'w')
+
+
+def insert_file_in_gap():
+    start_number = 0
+    for filename in sorted(os.listdir(TEST_FILES_DIRECTORY)):
+        if re.match(r"{0}\d{{3}}".format(FILE_NAME), filename):
+            fileNumRegex = re.compile(r'(\d)[^\d]*$')
```
Comment (gradopaw) [inline students/saleta_sebastian/lesson_10_organizing_files/insert_the_gaps.py]: Wrong variable name

File: students/saleta_sebastian/lesson_10_organizing_files/filling_in_the_gaps.py (hunk 1 above)
Comment (gradopaw) [inline students/saleta_sebastian/lesson_10_organizing_files/filling_in_the_gaps.py]: I have a bit modified your test data:
```for i in random.sample(range(2, 20), 5):``` and as I can see results are wrong 😄 Could you have look at this one more time?

File: students/saleta_sebastian/lesson_10_organizing_files/insert_the_gaps.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+TEST_FILES_DIRECTORY = 'insert_gaps_files'
+FILE_NAME = 'spam'
+TXT_EXTENSION = 'txt'
+
+
+def create_test_data():
+    if not os.path.exists(TEST_FILES_DIRECTORY):
+        os.makedirs(TEST_FILES_DIRECTORY)
+    # Create files with gaps
+    for i in range(2, 9):
```
Comment (gradopaw) [inline students/saleta_sebastian/lesson_10_organizing_files/insert_the_gaps.py]: I have a bit modified your test data:
```for i in random.sample(range(2, 20), 5):``` and as I can see results are wrong 😄 Could you have look at this one more time?

File: students/saleta_sebastian/lesson_10_organizing_files/filling_in_the_gaps.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        open(os.path.join(TEST_FILES_DIRECTORY, "{}{:03d}.{}".
+                          format(FILE_NAME, i, TXT_EXTENSION)), 'w')
+
+
+def fill_the_gaps():
+    start_number = 0
+    for filename in sorted(os.listdir(TEST_FILES_DIRECTORY)):
+        if re.match(r"{0}\d{{3}}".format(FILE_NAME), filename):
+            fileNumRegex = re.compile(r'(\d)[^\d]*$')
```
Comment (gradopaw) [inline students/saleta_sebastian/lesson_10_organizing_files/filling_in_the_gaps.py]: According to PEP8 variable names should be lowercase with underscore

**Comment by ssaleta** [comment]
@gradopaw  👍

**Pull request merged**